styles/components/purposes.css

```diff
@@ -0,0 +1,45 @@
+.purposes__row {
+  display: flex;
+  justify-content: space-between;
+}
+
+.purpose {
+  background: var(--white);
+  box-shadow: var(--boxs);
+  display: flex;
+  align-items: center;
+  flex-shrink: 0;
+  width: calc((100% - 40px) / 2);
```

Comment:
This is style for mobile for desktop? I saw desktop design, we have 3 cols not 2 cols